### PR TITLE
TWIS-80/user-account-section-icon

### PR DESCRIPTION
**Pull request by Jack-Vu**
# TWIS-80/user-account-section-icon

## Link to Story

https://re-boot.atlassian.net/browse/TWIS-80

## Description

Please include a summary of the change and which issue is fixed.

## Screenshot(s) / GIF(s):
Desktop:
image

Tablet:
image

Mobile:
image

## Type of change

- [ ] Bug fix
- [x] New feature
- [ ] Dependency / Configuration update
- [ ] Documentation update

## Checklist:

- [x] My code follows the project's style guidelines
- [x] I have created a Storybook story for my component(s)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by ShinK8245 — DISMISSED** [review]

File: src/app/_components/UserIcon/UserIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+            </Popper>
+          )}
+        </Box>
+      </ClickAwayListener>
+      {/* placed the drawer outsize of the click away listener because it was causing issues with how drawer was closing */}
+      {isMobileSmallScreen && (
+        <Drawer
+          open={open}
+          onClose={toggleDrawer(false)}
+          sx={{ width: "200px", maxWidth: "200px" }}
```
Comment (Muradil-Erkin) [inline src/app/_components/UserIcon/UserIcon.tsx:141]: Don't need a drawer for mobile, i think we mentioned in the story that mobile will use the same popper that desktop has.

**Review by ShinK8245 — APPROVED** [review]
LGTM!

**Review by SlowGti — APPROVED** [review]
LGTM

**Comment by Jack-Vu** [comment]
@SlowGti @ShinK8245 fixed conflict mind re-reviewing this pr

**Pull request merged**